dracut/30ignition/ignition-apply-kargs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+# Checks if kernel argument directory exists,
+# then redeploy and reboot the system if it exists
+reboot_if_kargs_dir_exists() {
+    local REFSPEC=( $(ostree refs --repo /sysroot/ostree/repo) )
+    if [ -d /sysroot/etc/ostree/kargs.d ]
+    then
+        ostree admin deploy REFSPEC
```

Comment:
You definitely need a `--sysroot=/sysroot` argument here.
And also it should be `$REFSPEC` and not `REFSPEC` literally.

Also there is `ostree admin instutil set-kargs` which mutates the deployment in-place rather than creating a new one which...is probably better here.

Reply:
Yes `--sysroot=/sysroot` solved the issue here : )

> And also it should be $REFSPEC and not REFSPEC literally.

I only corrected this in local testing branch, should've updated pr branch too. Sorry about this.
Currently I'm testing on `ostree admin instutil set-kargs` :+1: